ebu_tt_live/ui/user_input_producer/uip.js

```diff
@@ -423,8 +385,10 @@ $(document).ready(function() {
     $("#scheduled-send-clock-selector").change(handleScheduledSendSetupDependingOptions);
 
     $("#synchronize-media-clock-button").click(function() {
-        scheduled_send_media_clock_offset = Date.now();
-        stopResetRunningClock();
+        scheduled_send_media_clock_offset = moment();
+        //stopResetRunningClock();
+        $('#running-clock').hide();
+        $('#running-clock-media').show();
         interval_running_clock = setInterval(updateRunningClockMedia, 500);
```

Comment:
On Friday, Zoltan explained to me that because there were too many `setInterval` instances, it was causing the browser to crash, so `setTimeout` was the better choice for the clocks.

I wasn't 100% sure what line 389 was doing, so I commented it out for that instance - I can put it back.